client/layout/masterbar/style.scss

```diff
@@ -489,3 +489,8 @@ $autobar-height: 20px;
 		transform: translateY( -48px );
 	}
 }
+
+.hide-masterbar .masterbar {
+	opacity: 0;
+	pointer-events: none;
```

Comment:
Thinking out loud-

`visibility: hidden` should cover both of these rules. It does not have a smooth transition which may or may not have a use case. Probably good to leave this as is.